squidb-processor/src/com/yahoo/squidb/processor/plugins/defaults/properties/generators/RowidPropertyGenerator.java

```diff
@@ -57,6 +58,9 @@ public TypeName getTypeForAccessors() {
     private void addAccessorDocumentationForRowids(MethodSpec.Builder params, boolean getter) {
         if (isUnaliasedRowid()) {
             params.addAnnotation(Override.class);
+            if (!getter) {
```

Comment:
No, because the getter in the unalised rowid case returns a primitive, so it doesn't need an annotation. The setter is the one that returns the table model object.